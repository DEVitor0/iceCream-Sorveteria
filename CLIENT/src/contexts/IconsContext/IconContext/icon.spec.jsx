import React from 'react';
import { render } from '@testing-library/react';
import IconContext from './index';

test('checks if IconContext is a valid context', () => {
  const { asFragment } = render(
    <IconContext.Provider value={{ icon: 'home' }}>
      <div />
    </IconContext.Provider>,
  );
  expect(asFragment()).toMatchSnapshot();
});
