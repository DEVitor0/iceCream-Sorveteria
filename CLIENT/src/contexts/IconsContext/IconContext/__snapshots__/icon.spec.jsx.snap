// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checks if IconContext is a valid context 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
